examples/demo.jl

```diff
@@ -15,7 +14,7 @@ using TransformersLite
 include("utilities.jl")
 include("training.jl")
 
-path = "datasets\\amazon_reviews_multi\\en\\1.0.0\\"
+path = joinpath(@__DIR__, "..", "datasets", "amazon_reviews_multi", "en", "1.0.0")
```

Comment:
The readme currently says change to this directory first, then the ".." won't be needed. So either this should be updated or the readme.